src/screens/HelpScreen.tsx

```diff
@@ -15,6 +15,18 @@ const createText = () => {
   });
 };
 
+const phoneCall = () => {
+  let phoneNumber = '';
+
+  if (Platform.OS === 'android') {
+    phoneNumber = 'tel:${1234567890}';
```

Comment:
Waarvoor is t ${}? Want normaal kan je dat in javascript gebruiken om makkelijk variabele in een string te doen, alleen dan moet je well \` \` gebruiken ipv ' '. Evt dat het telefoon nummer nog als const ergens kan?

Reply:
whoops foutje